docs/Account.md

```diff
@@ -24,6 +24,11 @@ A more detailed writeup on the topic can be found on [Perama's blogpost](https:/
   * [`set_public_key`](#set_public_key)
   * [`is_valid_signature`](#is_valid_signature)
   * [`__execute__`](#__execute__)
+  * [`is_valid_eth_signature`](#is_valid_eth_signature)
+  * [`eth_execute`](#eth_execute)
+  * [`_unsafe_execute`](#_unsafe_execute)
+* [Presets](#presets)
+  * [Eth Account](#eth-account)
```

Comment:
Good call adding these in! We should add headers for `Presets` and `Eth Account` in the docs (the links don't point to anything right now). I think our vanilla `Account` should be in `Presets` as well